### PR TITLE
add Metaverse ETP

### PR DESCRIPTION
**Pull request by ehsueh**
add Metaverse ETP

### CODE REVIEW

File: common/defs/bitcoin/etp.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 600,
+  "uri_prefix": "etp",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [],
+  "blockbook": [
+    "https://explorer.mvs.org/api"
```
Comment (prusnak) [inline common/defs/bitcoin/etp.json]: This is not a blockbook. Remove this.

**Comment by prusnak** [comment]
Which wallet do you plan to add Trezor integration to?

File: common/defs/bitcoin/etp.json (hunk 1 above)
Comment (ehsueh) [reply common/defs/bitcoin/etp.json]: @prusnak Oh! I see. Thanks! I've removed it and updated the icon now it's passing the format check. However, I'm getting an error saying `Files src/trezorlib/coins.json and /tmp/tmp.U8aWo9Dv2W differ`. How do I fix this? Also, should I modify other files such as support.json?

**Comment by ehsueh** [comment]
@prusnak We are looking at adding it to our wallet at myetpwallet.com

**Comment by prusnak** [comment]
You need to run `make templates`

**Comment by ehsueh** [comment]
Ok ran `make templates`. I had troubles running it on a WSL Debian (do you know why that might be?) but was able to do it on a native ubuntu. 
I had to install some dependencies
```
pip3 install setuptools wheel trezor mako munch
```
@prusnak Do you think it would be possible to add this step along with the dependencies in `README.md` or on the dev guide here https://wiki.trezor.io/Developers_guide-Contributing#Adding_a_new_coin?

**Comment by ehsueh** [comment]
@prusnak  Could you have a look at the new error? Thanks!

**Comment by matejcik** [comment]
@ehsueh make sure you have submodules; `git submodule update --init --recursive`
then rerun `make gen`

**Comment by ehsueh** [comment]
@matejcik Thanks! I did `git submodule update --init --recursive` then `make gen` and rerun `make templates`. 😃  Additional dependency is installed from `https://github.com/protocolbuffers/protobuf/releases/` for whomever came across similar issues in the future. Do I just wait for trezor's team to merge the new `coins.json` differences now?

**Comment by prusnak** [comment]
@ehsueh Which wallet Metaverse ETP has committed to integrate Trezor so the coin support is actually usable?

**Comment by ehsueh** [comment]
@prusnak None yet but the plan is to integrate it here (myetpwallet.com)

**Comment by ehsueh** [comment]
@prusnak Where can I find guides or example projects that show how to integrate Trezor to an existing  wallet?

**Comment by ehsueh** [comment]
@prusnak We are trying to get integrated on Magnum Wallet but told that we need ETP supported by Trezor first before Trezor integration can happen on Magnum Wallet. Could you help us merge the ETP changes? Thank you!

**Comment by prusnak** [comment]
Added in ac201fa92b8afabd7e61b5ed5fbda836a3bbea39

**Pull request closed without merging**